group/advanced/add.md

```diff
@@ -40,6 +40,12 @@ Create a new **group**.
 #### ❌ Raises
 * `SecurityRequestError`<br>
   Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `AddOperationError`<br>
+  Raises `AddOperationError` when the **group** cannot be added because it already exists in the environment.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** supplied contains at least one unknown trait.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** supplied contains at least one unknown trait.
```

Comment:
`SegmentTraitError` should only be documented once here.

Reply:
Nice catch, I must have been trigger happy with copy/paste. It is resolved now.